app/models/student_loans/dqt_record.rb

```diff
@@ -23,11 +23,11 @@ class DqtRecord
     # :qts_date - The date the teacher achieved qualified teacher status.
     #             Format: %d/%m/%Y
     def initialize(record, _ = nil)
-      @qts_award_date = record.fetch(:qts_date)
-      @degree_codes = record.fetch(:degree_codes, ["ANY"])
-      @itt_start_date = record.fetch(:itt_date, nil)
-      @itt_subject_codes = record.fetch(:itt_subject_codes, ["ANY"])
-      @qualification_name = record.fetch(:qualification_name, nil)
+      @qts_award_date = record.qts_date)
+      @degree_codes = record.degree_codes || ["ANY"]
+      @itt_start_date = record.itt_date
+      @itt_subject_codes = record.itt_subject_codes || ["ANY"]
```

Comment:
@WorkSutton who's best to ask about the behaviour here? at the moment, the new API client would return: `[nil, nil, nil]` if there were no values for the subject codes. I'm wondering if I should compress to get rid of the nils, then in this portion of the code run `.presence` and supply `["ANY"]` if it returns `nil`